Rakefile

```diff
@@ -1,7 +1,23 @@
 ENV['SINATRA_ENV'] ||= 'development'
 
+require 'sinatra/activerecord/rake'
+require 'securerandom'
+
+namespace :db do
+  task :load_config do
+    require './app/controllers/application_controller'
+  end
+
+  desc 'Generate auth token for the application'
+  task :generate_token do
+    token = SecureRandom.urlsafe_base64
+    AuthToken.delete_all
```

Comment:
Should deleting all tokens be in a separate task?

Reply:
No, I figured for the first iteration, we only want 1 token for security and only ever one token. At least until/unless we have actual separate users. This functions the same way may API tokens function. It will display the token once generated, then if the user loses it, they regenerate.